### PR TITLE
Create dev branch

### PR DESCRIPTION
**Pull request by koomekevo**
This Pull Request has been created under dev branch to achieve the following:
- Create React App.
- Edit initial README.md information.
- Add Stylelint and EsLint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,16 @@
```
Comment (Meltrust) [inline README.md:4]: - [ ] OPTIONAL: If you add a screenshot of your project to your readme, your project documentation will look a lot better ✔️

File: package.json [hunk 2]
```diff
@@ -6,12 +6,16 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
```
Comment (Meltrust) [inline package.json:7]: - [ ] Please, kindly add tests for your app.  The only test I could find is the one that comes with `create-react-app`.

image

image


That way, you will ensure the scalability of your app while making sure it still works as intended.

File: src/pages/MissionsList.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            const bdj = badgeClass(mission.joined);
+            let memberStatus;
+            let memberAction;
+            if (mission.joined) {
+              memberAction = 'Leave mission';
+              memberStatus = 'Active member';
+            } else {
+              memberAction = 'Join mission';
+              memberStatus = 'NOT A MEMBER';
+            }
```
Comment (Meltrust) [inline src/pages/MissionsList.js]: - [ ] For this particular project, the React conditional render syntax is required for conditionally displaying both the Rockets and the Missions:

image

Don't worry! I'll show you how!! 💪 💪 

This is an example:

image

Another example with a kind of "if/else" scheme:

image


The rule is that if the condition on the left of the `&&` is true, the action of the right will be run. 💪 

Now that you know how to do it kindly please implement this syntax to your missions and rockets conditional rendering (i.e. for the "reserved" badge, the "join" badge, etc). ✔️

File: src/pages/RocketsCards.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+              <button
+                type="button"
+                title="button"
+                className="rockets-desc-btn rocktbtn"
+                onClick={reserveRockets}
+                id={rocket.id}
+              >
+                Reserve Rocket
+              </button>
+            )}
```
Comment (Meltrust) [inline src/pages/RocketsCards.js]: _(This is the second iteration of my last comment about the conditional rendering syntax.  Copied it here just for reference.  It's exactly the same requirement as with the rockets)._

- [ ] For this particular project, the React conditional render syntax is required for conditionally displaying both the Rockets and the Missions:

image

Don't worry! I'll show you how!! 💪 💪 

This is an example:

image

Another example with a kind of "if/else" scheme:

image


The rule is that if the condition on the left of the `&&` is true, the action of the right will be run. 💪 

Now that you know how to do it kindly please implement this syntax to your missions and rockets conditional rendering (i.e. for the "reserved" badge, the "join" badge, etc). ✔️

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @mohamedelshamy55, 


There are some issues that you still need to work on to go to the next project but you are almost there!

## You did a great job here team, the project works flawlessly! However, in order to comply with 2 of the Microverse requirements, we need to make an addition and some small changes.  Kindly please look at my comments below 👇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @koomekevo and @mohamedelshamy55, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## STATUS APPROVED ✅ 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**